pkg/cache/multi_namespace_cache.go

```diff
@@ -99,10 +112,24 @@ func (c *multiNamespaceCache) GetInformerForKind(ctx context.Context, gvk schema
 		}
 		informers[ns] = informer
 	}
+	clusterCacheInf, err := c.clusterCache.GetInformerForKind(ctx, gvk)
```

Comment:
Same here, we shouldn't call this unless the object is cluster-scopped (and then we should _only_ call this)